src/proxy-ui-api/frontend/src/util/helpers.ts

```diff
@@ -41,3 +41,30 @@ export function isValidRestURL(str: string) {
   return isValidWsdlURL(str);
 }
 
+// Save response data as a file
+export function saveAsFile(response: any) {
```

Comment:
This function is provided as a global utility saveAsFile, but the implementation doesn't seem to be very usefull in many situations, right? I'd try to come up with a more specific name if that is the case.

Reply:
Now has a slightly better name